compiler/modules/front/expr.bal

```diff
@@ -587,24 +591,36 @@ function codeGenLogicalBinaryExpr(ExprContext cx, bir:BasicBlock bb, s:BinaryLog
     var [leftValue, leftFlags] = booleanOperandValue(lhs);
     if leftFlags != 0 {
         if leftValue == isOr {
-            // XXX not correct errors in `right` won't be found
-            // XXX previous code used bb not block1 here
+            bir:BasicBlock fakeBlock = cx.createBasicBlock();
```

Comment:
This isn't going to work in a const definition (because cx.createBasicBlock with create an error).